src/main/java/me/moodcat/api/models/SongModel.java

```diff
@@ -73,6 +73,24 @@
     private double arousal;
 
     /**
+     * The link to the site to purchase the song.
+     *
+     * @param purchaseUrl
+     *          The new purchaseURL to set.
```

Comment:
I'm not sure about this one, but I think it is out of scope of this PR as it affects a lot of Javadoc that doesn't have to do with the purchase URL.